optimum/exporters/onnx/convert.py

```diff
@@ -370,7 +370,9 @@ def export_tensorflow(
     output_names = list(config.outputs.keys())
 
     config.patch_ops()
-    input_signature = [tf.TensorSpec.from_tensor(tensor, name=key) for key, tensor in dummy_inputs.items()]
+    input_signature = [
+        tf.TensorSpec([None] * tensor.ndim, dtype=tensor.dtype, name=key) for key, tensor in dummy_inputs.items()
```

Comment:
It means that we do not specify the shape for any dimension?

Reply:
Yes, but not sure if it's the best way. I added this because of a failing test. Transformers [transformers:convert.py#LL279](https://github.com/huggingface/transformers/blob/main/src/transformers/onnx/convert.py#LL279) also uses the same method to export.

Could try to augment it with the `config.inputs` dynamic_axes.

Reply:
Updated, only the axes having dynamic shapes will be updated with None.

PS. Probably a separate issue, the tf export do not work when `past_key_values` are in the input.